project/package/CodeEditor.vue

```diff
@@ -386,6 +390,21 @@ export default {
   margin: 0;
 }
 
+count-lines{
+  counter-reset: line;
+}
+.count-lines:deep(span.hljs-tag){
+  counter-increment: line;
+}
+.count-lines:deep(span.hljs-tag:before){
+  content: counter(line);
+  position: absolute;
+  left: 5px;
+  font-size: 8px;
+  color: #777777;
+  padding-top: 4px;
```

Comment:
Perhaps add padding-right: 2px? So there is a little bit of room between the numbers and the code.

Reply:
The padding-right would have no effect, what is possible is decrease the font-size to 7px and set left to 1px in this way you would have more space

Look the screenshots

![Schermata 2022-10-18 alle 11 44 24](https://user-images.githubusercontent.com/36926081/196396339-9a34d7ac-8034-4ad8-a78b-3a1a79d01b32.png)